packages/api-generator/src/map.js

```diff
@@ -390,6 +406,194 @@ module.exports = {
       }
     ]
   },
+  'v-calendar': {
+    scopedSlots: [
+      {
+        name: 'day',
+        props: VTimestampScopedProps
+      },
+      {
+        name: 'dayBody',
+        props: VTimestampScopedProps
+      },
+      {
+        name: 'dayHeader',
+        props: VTimestampScopedProps
+      },
+      {
+        name: 'dayLabel',
+        props: VTimestampScopedProps
+      },
+      {
+        name: 'dayMonth',
+        props: VTimestampScopedProps
+      },
+      {
+        name: 'interval',
+        props: VTimestampScopedProps
+      }
+    ],
+    functions: [
+      {
+        name: 'weekdayFormat',
+        signature: '(timestamp: {date: string, time: string, year: number, month: number, day: number, hour: number, minute: number, weekday: number, hasDay: boolean, hasTime: boolean, past: boolean, present: boolean, future: boolean}, short: boolean): string'
```

Comment:
Is signature identical for these? If so pull it out into a variable.

Reply:
@nekosaur looks like not